### ISSUE TITLE
There is a problem about toc could only create toc table once.

### ISSUE THREAD

**Issue by gwdgithubnom** [nok/markdown-toc#80]
Os: windows 10
Atom version: 1.10.0
Issues:  It could only create toc table only once, and for next, you should disable it and then enable it. it could work again. So, when i need create a toc table, I need disable it then enable it.

**Comment by lanceplaine** [comment]
Seems like a duplicate of #54

**Comment by gwdgithubnom** [comment]
@lanceplaine , I had see #54 issue. Then tried did as you said, but it didn't like that. It's only work once, then it could not work again only if you restart atom or disable it firstly  then enable it. Here is a micro-video to describe it.
![bug describe](https://github.com/gwdgithubnom/issues-resources/blob/master/nok/markdown-toc/2016/ato-markdown-toc-did-not-work.gif)

**Comment by dkaksl** [comment]
I'm having this same issue.

App                 | Version
:--------------- | :----
atom                | 1.15.0
markdown-toc | 0.4.2

**Comment by haozan** [comment]
I'm having this same issue.

**Comment by lenarddome** [comment]
+1

**Comment by barcia** [comment]
I have the same issue in KDE Neon (Ubuntu 16.04)

Software | Version 
--- | ---: |
Atom | 1.17.2 x64
markdown-toc | 0.4.2

**Comment by paulplayer** [comment]
+1
